src/witan/models/dem/ccm/mort/mortality.clj

```diff
@@ -55,75 +55,106 @@
+  docs)
+  Or
+  Takes a dataset with projected death rates for the future, a dataset with
   historic mortality rates, and parameters for: first projection year, last
   projection year, the scenario to use in the future rates dataset (e.g. :low,
   :principal or :high), the start and end  years of data to average for the calculation,
   and the last year of mortality data. Returns a dataset that now includes projected
   mortality rates, projected with the jumpoff year average method combined with an
   applied national trend (see docs)"
-  {:witan/name :ccm-mort/project-asmr-average-applynationaltrend
-   :witan/version "1.0.0"
+  {:witan/name :ccm-mort/project-asmr
+   :witan/version "1.1.0"
    :witan/input-schema {:historic-asmr HistASMRSchema
                         :future-mortality-trend-assumption NationalTrendsSchema}
-   :witan/param-schema {:start-yr-avg-mort (s/constrained s/Int m-utils/year?)
-                        :end-yr-avg-mort (s/constrained s/Int m-utils/year?)
+   :witan/param-schema {:start-yr-avg-mort s/Int
+                        :end-yr-avg-mort s/Int
+                        :variant (s/enum :fixed-average :fixed-trend
+                                         :ant-average :ant-trend)
                         :first-proj-yr (s/constrained s/Int m-utils/year?)
                         :last-proj-yr (s/constrained s/Int m-utils/year?)
                         :mort-scenario (s/enum :low :principal :high)
                         :mort-last-yr (s/constrained s/Int m-utils/year?)}
    :witan/output-schema {:initial-projected-mortality-rates ProjASMRSchema}
-   :witan/exported? false}
+   :witan/exported? true}
   [{:keys [historic-asmr future-mortality-trend-assumption]}
-   {:keys [start-yr-avg-mort end-yr-avg-mort first-proj-yr
+   {:keys [start-yr-avg-mort end-yr-avg-mort variant first-proj-yr
            last-proj-yr mort-scenario mort-last-yr]}]
-  (let [projected-rates-jumpoffyr (cf/jumpoffyr-method-average historic-asmr
-                                                               :death-rate
-                                                               :death-rate
-                                                               start-yr-avg-mort
-                                                               end-yr-avg-mort)]
-    {:initial-projected-mortality-rates (cf/apply-national-trend projected-rates-jumpoffyr
-                                                                 future-mortality-trend-assumption
-                                                                 first-proj-yr
-                                                                 last-proj-yr
-                                                                 mort-scenario
-                                                                 :death-rate)}))
+  (let [asmr-projection-method (case variant
+                                 :fixed-average {:initial-projected-mortality-rates
```

Comment:
could you swop the naming of the variants around please - its a small thing, but its a logic thing as the jump off year method comes first